<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>256</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>256</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>256</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Umut Tarlaları Şifre Oluşturucu</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget[objectName*=&quot;centra&quot;] {
	background-image: url(:/Images/umut_tarlalari_02.png);
}

QSpinBox {
	background-color: rgba(255, 255, 255, 64);
    selection-background-color: #00000000;
	selection-color: rgb(45, 255, 19);
	color: rgb(255, 233, 62);
    border: 2px solid blue;
    border-radius: 10px;
    padding-left: 0px;
    padding-top: 0px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="kod1">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>10</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>45</number>
    </property>
   </widget>
   <widget class="QLabel" name="key">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>231</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(177, 122, 45)</string>
    </property>
    <property name="text">
     <string>12</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="kod2">
    <property name="geometry">
     <rect>
      <x>116</x>
      <y>10</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>45</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="kod3">
    <property name="geometry">
     <rect>
      <x>219</x>
      <y>10</y>
      <width>91</width>
      <height>71</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>45</number>
    </property>
   </widget>
   <widget class="QPushButton" name="aboutButton">
    <property name="geometry">
     <rect>
      <x>284</x>
      <y>222</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>28</width>
      <height>28</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/Images/Information_icon.svg</normaloff>:/Images/Information_icon.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
